.github/CODEOWNERS

```diff
@@ -19,6 +19,26 @@ runtime/autoload/tar.vim		@cecamp
 runtime/autoload/vimball.vim		@cecamp
 runtime/autoload/xmlformat.vim		@chrisbra
 runtime/autoload/zip.vim		@cecamp
+runtime/colors/tools/check_colors.vim	@habamax @romainl @neutaaaaan
+runtime/colors/blue.vim			@habamax @romainl @neutaaaaan
+runtime/colors/darkblue.vim		@habamax @romainl @neutaaaaan
+runtime/colors/default.vim		@habamax @romainl @neutaaaaan
+runtime/colors/delek.vim		@habamax @romainl @neutaaaaan
+runtime/colors/desert.vim		@habamax @romainl @neutaaaaan
+runtime/colors/elflord.vim		@habamax @romainl @neutaaaaan
+runtime/colors/evening.vim		@habamax @romainl @neutaaaaan
+runtime/colors/industry.vim		@habamax @romainl @neutaaaaan
+runtime/colors/koehler.vim		@habamax @romainl @neutaaaaan
+runtime/colors/morning.vim		@habamax @romainl @neutaaaaan
+runtime/colors/murphy.vim		@habamax @romainl @neutaaaaan
+runtime/colors/pablo.vim		@habamax @romainl @neutaaaaan
+runtime/colors/peachpuff.vim		@habamax @romainl @neutaaaaan
+runtime/colors/ron.vim			@habamax @romainl @neutaaaaan
+runtime/colors/shine.vim		@habamax @romainl @neutaaaaan
+runtime/colors/slate.vim		@habamax @romainl @neutaaaaan
+runtime/colors/tools			@habamax @romainl @neutaaaaan
```

Comment:
Maybe one of line 22 or line 39 can be removed.

Reply:
Fixed in #[236442a](https://github.com/vim/vim/pull/9795/commits/236442aeac91f56326ff5164713dd2569ccc792c).